tests/model_serving/model_server/private_endpoint/constants.py

```diff
@@ -0,0 +1,33 @@
+from typing import Dict, List, Any
+
+AWS_REGION: str = "us-east-2"
+AWS_BUCKET: str = "ods-ci-wisdom"
+AWS_ENDPOINT: str = "https://s3.us-east-2.amazonaws.com/"
+
+SR_CONTAINERS_KSERVE_CAIKIT: List[Dict[Any, Any]] = [
+    {
+        "args": ["--model-name=/mnt/models/artifacts/"],
+        "command": ["text-generation-launcher"],
+        "env": [{"name": "HF_HOME", "value": "/tmp/hf_home"}],
+        "image": "quay.io/modh/text-generation-inference@sha256:294f07b2a94a223a18e559d497a79cac53bf7893f36cfc6c995475b6e431bcfe",
```

Comment:
This is what i was thinking.  I would suggest implementing an action that can download the `.env` file from the model-controller repository based on the branch, copy it to a specified folder in this repo, and make it available for testing using constants. Additionally, plan to add an option for users to provide an image URL as input for their tests. This would override the values in the constants file if not provided use the image present in the env file), allowing us to run tests on any image dynamically with input variables.

Reply:
iiuc in this use case you can use the template.
we should have the info in the cluster and avoid setting this values in the tests